templates/terraform-module-page.md

```diff
@@ -0,0 +1,85 @@
+{{- defineDatasource "config" .Env.README_YAML | regexp.Replace ".*" "" -}}
```

Comment:
Add a comment to the top of the output indicating that the file was programatically generated from the `REAMDE.md`; ideally, with a link to the file. 

See the comment atop this file: 
https://raw.githubusercontent.com/cloudposse/build-harness/master/templates/README.md